internal/api/cluster_installation.go

```diff
@@ -533,9 +541,25 @@ func handleDeleteInActiveClusterInstallationsByCluster(c *Context, w http.Respon
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
-
 	c.Logger.Infof("Successfully deleted inactive cluster installations for cluster ID: %s", clusterID)
+
+	IsActive := false
+	filter := &model.ClusterInstallationFilter{
+		ClusterID: clusterID,
+		IsActive:  &IsActive,
+		Paging:    model.AllPagesNotDeleted(),
+	}
+	deletedClusterInstallations, err := c.Store.GetClusterInstallations(filter)
```

Comment:
Although unlikely, there is a possible race condition as we only fetch not deleted Installations. I think we could get just the count of those Installations in `DeleteInActiveClusterInstallationByClusterID` (as part of the transaction before deleting) or change the response.